tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -667,6 +672,31 @@ class TestGetTaskInstances(TestTaskInstanceEndpoint):
                 2,
                 id="test queue filter ~",
             ),
+            pytest.param(
+                [
+                    {"executor": "test_exec_1"},
+                    {"executor": "test_exec_2"},
+                    {"executor": "test_exec_3"},
```

Comment:
I see that you don't use test_exec_3 anywhere later, was that intentional?

Reply:
Yeah, the other tests do the same. It's to ensure the filter is actually working and not just returning all items (so we create three and query for a subset of them).